doc/sphinx/proof-engine/ltac.rst

```diff
@@ -896,6 +921,7 @@ produce subgoals but generates a term to be used in tactic expressions:
    value of :n:`@ident` by the value of :n:`@expr`.
 
    .. exn:: Not a context variable.
+   .. exn:: Unbound context identifier @ident.
```

Comment:
Would be great if you could add `:undocumented:` options to these two `exn`.

Reply:
It looks like the `Not a context variable` one is not even implemented actually.
```coq
Goal True.
let ident := idtac in
let x := context ident [O] in x.
(* Anomaly "Uncaught exception Ltac_plugin.Taccoerce.CannotCoerceTo("a term context")." *)
```